deps/build.jl

```diff
@@ -2,14 +2,58 @@
 import JSON
 using Compat
 
+# remove deps.jl if it exists, in case build.jl fails
+isfile("deps.jl") && rm("deps.jl")
+
+if isdir(abspath(dirname(@__FILE__), "..", "..", "Conda"))
```

Comment:
Ok. My thought was actually to detect if Conda.jl was installed else use system python. This is also the reason why I did not add Conda.jl to `REQUIRE`. If you think it is appropiat to add Conda as dependency i will do it.